src/plugins/tool_extrude.py

```diff
@@ -109,21 +129,33 @@ def on_drag(self, target):
 
         if ty != 0 or tx != 0 or tz != 0:
             self._mouse = (target.mouse_x, target.mouse_y)
```

Comment:
should be only reset if we actual draw a stamp and the if can be replaced by an early return if dx and dy equals 0